pkg/kapp/app/recorded_app.go

```diff
@@ -23,8 +23,10 @@ const (
 )
 
 type RecordedApp struct {
-	name   string
-	nsName string
+	name              string
+	nsName            string
+	creationTimestamp time.Time
+	labels            map[string]string
```

Comment:
what uses labels?

Reply:
For setting `app.ObjectMeta.Labels` value in `labels` for recordedApp `recordedApp := &RecordedApp{app.Name, app.Namespace, app.ObjectMeta.Labels, a.coreClient...`